### PR TITLE
Feature/paint 003 Merge into develop

### PR DESCRIPTION
**Pull request by uwe-h**

### CODE REVIEW

File: app/src/androidTest/java/at/tugraz/ist/swe/DrawLineTest.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                {
+                    FlyoutMenuView.MenuItem result = activityTestRule.getActivity().toolFlyoutMenu.getAdapter().getItem(counter);
+                    if(((FlyoutToolbar.MenuItemImage)result).getID() == R.drawable.ic_si_glyph_line_two_angle_point)
+                    {
+                        activityTestRule.getActivity().toolFlyoutMenu.setSelectedMenuItem(result);
+                        break;
+                    }
+                }
+                PaintingTool tool = activityTestRule.getActivity().drawingArea.getPaintingTool();
+                Log.d("TEST", tool.toString());
```
Comment (MasaruKondera) [inline app/src/androidTest/java/at/tugraz/ist/swe/DrawLineTest.java]: Suggesting to remove the log. Not needed in a release version.

Comment (uwe-h) [reply app/src/androidTest/java/at/tugraz/ist/swe/DrawLineTest.java]: Done.

File: app/src/main/java/at/tugraz/ist/swe/DrawArea.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import android.graphics.Bitmap;
 import android.graphics.Canvas;
 import android.graphics.Color;
+import android.util.Log;
```
Comment (MasaruKondera) [inline app/src/main/java/at/tugraz/ist/swe/DrawArea.java]: Delete log import and Log line at line ~37

File: app/src/main/java/at/tugraz/ist/swe/Line.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package at.tugraz.ist.swe;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Point;
+import android.util.Log;
```
Comment (MasaruKondera) [inline app/src/main/java/at/tugraz/ist/swe/Line.java]: Delete unused Log import.

**Pull request merged**